Automattic/wp-calypso
# Update TwitterPreview to make hashtags and URLs clickable

Related to https://github.com/Automattic/jetpack/pull/30029

## Proposed Changes

* Update TwitterPreview to make hashtags and URLs clickable

## Testing Instructions

* Follow the instructions given in https://github.com/Automattic/jetpack/pull/30029
* Add a custom message with hashtags and/or URLs
* Confirm that the hashtags and URLs are converted to hyperlinks

## Pre-merge Checklist





---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204281809108915
